### ISSUE TITLE
[Bulk add patient] Close toast button accessible text is not descriptive

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#5832]
## Area or Component

[Import patients from spreadsheet](https://demo.simplereport.gov/app/upload-patients?facility=f807754a-429e-47d6-91df-495684e161c0)

## Quick Summary

The success/error message has a `X` button to close the message, but the accessible text is just "close".

## Accessibility Violation

[Aria name is missing or incorrect](https://docs.deque.com/issue-help/1.0.0/en/aria-name-missing-incorrect)

## Action Requested

- Update the aria label for the close button to be more descriptive of what is closed. 

## Screenshots (optional)
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/f0e393b3-96c1-4c80-8c4a-d723875adca8)
## Original Deque Issue

https://cdc-devtools.dequecloud.com/issues/0dfb011b-a64a-42c4-8484-7a448c794037

**Comment by mehansen** [comment]
Closing as won't do
We feel 'close' accessible text is sufficiently descriptive
